networking/ingress-sharding.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="ingress-sharding"]
+= Ingress sharding in {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: ingress-sharding
+
+toc::[]
+
+In {product-title}, an Ingress Controller can serve all routes, or it can serve a subset of routes. By default, the Ingress Controller will serve any route created in any namespace in the cluster. Additional Ingress Controllers can be added to your cluster to optimize routing by creating _shards_, which are subsets of routes based on selected characteristics. To mark a route as a member of a shard, use labels in its `metadata` field. The Ingress Controller uses _selectors_, also known as a _selection expression_, to select a subset of routes from the entire pool of routes to serve.
```

Comment:
```suggestion
In {product-title}, an Ingress Controller can serve all routes, or it can serve a subset of routes. By default, the Ingress Controller will serve any route created in any namespace in the cluster. Additional Ingress Controllers can be added to your cluster to optimize routing by creating _shards_, which are subsets of routes based on selected characteristics. To mark a route as a member of a shard, use labels in the route or namespace `metadata` field. The Ingress Controller uses _selectors_, also known as a _selection expression_, to select a subset of routes from the entire pool of routes to serve.
```